### PR TITLE
Some minor fixes for configure scripts and add arm64 image mapping

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: scripts/image-builder/configure.sh [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
-sudo dnf install -y git osbuild-composer composer-cli \
-    cockpit-composer bash-completion podman genisoimage \
-    createrepo syslinux yum-utils selinux-policy-devel jq wget lorax rpm-build
+pkg_list="git osbuild-composer composer-cli \
+	cockpit-composer bash-completion podman genisoimage \
+	createrepo yum-utils selinux-policy-devel jq wget lorax rpm-build"
+
+if [ $(uname -p) != "aarch64" ]; then
+    pkg_list="${pkg_list} syslinux"
+fi
```
Comment (ggiguash) [inline scripts/image-builder/configure.sh:10]: We should be able to simply remove syslinux package from the list on all platforms. It'll be pulled in by lorax when necessary.

File: scripts/image-builder/configure.sh [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
+
+sudo dnf install -y ${pkg_list}
 sudo systemctl enable osbuild-composer.socket --now
 sudo systemctl enable cockpit.socket --now
-sudo firewall-cmd -q --add-service=cockpit
-sudo firewall-cmd -q --add-service=cockpit --permanent
+sudo firewall-cmd --add-service=cockpit
+sudo firewall-cmd --add-service=cockpit --permanent
```
Comment (ggiguash) [inline scripts/image-builder/configure.sh:16]: Is there no -q option on aarch64? Why remove it?

Comment (ggiguash) [reply scripts/image-builder/configure.sh:16]: Also, let's remove the duplicate command on line 15.

**Review by ggiguash — CHANGES_REQUESTED** [review]

File: scripts/image-builder/configure.sh (hunk 1 above)
Comment (praveenkumar) [reply scripts/image-builder/configure.sh]: @ggiguash looks like you already removed it.

**Pull request merged**